app/src/main/res/raw/init.js

```diff
@@ -61,3 +56,4 @@ window.web3.eth.getCoinbase = function(cb) {
     return cb(null, addressHex)
 }
 window.web3.eth.defaultAccount = addressHex
+window.web3.currentProvider.isAlphaWallet = false
```

Comment:
We should probably set `isAlphaWallet` to `true` instead. Right @James-Sangalli  ?(iOS sets it to false too).

```
web3.currentProvider.isAlphaWallet = true
```

Reply:
Yes, I thought the same thing.

Also, I notice we set isNeuron=true in the main provider code.

Maybe move the isAlphaWallet out into the main code and delete from the init code.

Reply:
I wonder if anyone is checking for alphawallet? If not I'll add a check on the office door, or at least I would if I was able to flash the firmware. I didn't add an OTA flash facility.

Reply:
So @JamesSmartCell , you can use this new commit for the updated provider which sets `isAlphaWallet` to `true`: https://github.com/AlphaWallet/AlphaWallet-web3-provider/commit/f25206c50009d1eb922c3cc8c0ba91594155e8b6